aspnetcore/native-aot/index.md

```diff
@@ -0,0 +1,180 @@
+---
+title: ASP.NET Core support for .NET native AOT
+author: mitchdenny
+description: Learn about ASP.NET Core support for .NET native AOT
+monikerRange: '>= aspnetcore-8.0'
+ms.author: midenn
+ms.custom: mvc
+ms.date: 04/10/2023
+uid: native-aot/index
+---
+# ASP.NET Core support for .NET native AOT.
+
+ASP.NET Core 8.0 introduces support for [.NET native ahead-of-time (AOT)](/dotnet/core/deploying/native-aot/). Not all ASP.NET Core features are compatible with native AOT at this time. 
+
+* How to get started.
+* The benefits and potential drawbacks.
+* The limitations.
+
+## Getting started with .NET native AOT deployment in ASP.NET Core
+
+When developing an app to be published using native AOT, a developer writes their application normally. Native AOT compilation occurs when an app is published, and the native AOT compiler analyses the code in the app to identify coding patterns incompatible with AOT.
```

Comment:
I know what you mean with "developer writes their application normally", but I'm wondering if we can word it differently.

```suggestion
Native AOT is a publishing option - the AOT compilation only happens when the app is published. A project opted into Native AOT publishing will still use JIT for F5 debugging, dotnet run, or dotnet build. There will be some observable differences though: some run time features that are not compatible with Native AOT will be disabled and throw exceptions at run time, and a source analyzer will be enabled to highlight code that is not compatible with Native AOT. At publish time, the entire app (including NuGets) will be analyzed for compatibility once more.
```

Reply:
I picked up Rick's language.